deploy/examples/common.yaml

```diff
@@ -13,6 +13,36 @@ metadata:
 ---
 kind: ClusterRole
 apiVersion: rbac.authorization.k8s.io/v1
+metadata:
```

Comment:
The NFS driver is really going to be rarely enabled. It's so rare that I would prefer that we not include the rbac in common.yaml. Instead, it would be nice to have an nfs-rbac.yaml. However, the reality of this may not be worth it since common.yaml is generated.

Reply:
I think that's a good point, and I like the idea of having a separate NFS RBAC. I can look into what it would take to generate `nfs-rbac.yaml` in a follow-up PR. It might not be too difficult.